PeerioTechnologies/peerio-mobile
# Fix android launcher icon

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/5246/mobile-fix-android-launcher-icons
#### Testing instructions  
Re-build the app and ensure that launcher icon is not clipped. Ideally this should be tested on 4 different resolution devices (I'm not sure about which device resolutions use which icon). But if it works on one device, it is safe to assume that it works on all.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
